### PR TITLE
Fix for #297 Printing release date multiple times

### LINKED ISSUE #297: Meetup date is printed loads of times for with javascript off (previously for a split second on page-load)
Found by @sarigriffiths 

Repro:

- Go to meetup.react.london

On page load you'll see this for a split second:

screen shot 2017-01-16 at 13 02 07

### PR DESCRIPTION
**Pull request by asavin**
giphy 12

Fixing sneaky statefulness in the ticket-status utility.

This fixes #297

### CODE REVIEW

File: shared/utilities/ticket-status/index.js [hunk 1]
```diff
@@ -64,8 +64,8 @@ export function getTicketStatusSubtitle(event, ticketStatusOptions) {
 }
 
 export function getTicketStatusOptions(event) {
-  const ticketStatusOptions = statusTypes[event.status];
```
Comment (asavin) [inline shared/utilities/ticket-status/index.js]: The root of the problem

File: shared/utilities/ticket-status/test.js [hunk 2]
```diff
@@ -9,13 +16,23 @@ describe('getTicketStatusOptions', () => {
       ticketReleaseDate: '2016-07-24T20:30:00+0000',
     };
     const result = getTicketStatusOptions(event);
-    expect(result).to.deep.equal({
```
Comment (asavin) [inline shared/utilities/ticket-status/test.js]: It works perfectly fine when called once, but if called multiple times it starts accumulating dates in the `title`

**Comment by dstodolny** [comment]
Nice one!
jackiechangivesathumbsupnumber1

File: shared/utilities/ticket-status/index.js [hunk 3]
```diff
@@ -64,8 +64,8 @@ export function getTicketStatusSubtitle(event, ticketStatusOptions) {
...
-  if (ticketStatusOptions) {
+  if (statusTypes[event.status]) {
+    const ticketStatusOptions = Object.assign({}, statusTypes[event.status]);
```
Comment (grahammendick) [inline shared/utilities/ticket-status/index.js]: @asavin You could destructure instead?

    const ticketStatusOptions = { ...statusTypes[event.status] };

**Pull request merged**